docs/source/tutorials/getting-started/transfer.rst

```diff
@@ -268,34 +289,39 @@ help you familiarize yourself with the library and with Daml interfaces.
 Split the Holding to Transfer the Right Amount
 ==============================================
 
-In the example, Bob requests ``USD 1,000`` from Alice and Alice allocates a holding for exactly the
+In the example, Bob requests ``USD 1'000`` from Alice and Alice allocates a holding for exactly the
```

Comment:
Can you revert this change? In English the notation is `USD 1,000` not `USD 1'000`

Reply:
Or maybe remove it altogether, USD 1000? I think that is how we have it in other parts of the docs, with one exception (1'000)

Reply:
I opened a separate issue to align this: https://github.com/digital-asset/daml-finance/issues/438

Reply:
missed your comment, good idea to open an issue